tasks/push-to-cdn/push-to-cdn.yaml

```diff
@@ -0,0 +1,110 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: push-to-cdn
+  labels:
+    app.kubernetes.io/version: "4.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task to push content to Red Hat's CDN
+  params:
+    - name: releasePlanAdmission
+      type: string
+      description: The namespaced name of the ReleasePlanAdmission
+      default: "rhtap-releng-tenant/rpa" # This needs to change later
+    - name: subdirectory
+      description: Subdirectory inside the workspace to be used
+      type: string
+      default: ""
+  workspaces:
+    - name: data
+      description: Workspace to save the CR jsons to
+  results:
+    - name: releasePlanAdmission
+      type: string
+      description: The relative path in the workspace to the stored releasePlanAdmission json
+  steps:
+    - name: create-exodus-conf-file
+      image: quay.io/distribution_relengs/publish-to-dev-portal-test-image
+      env:
+        - name: "RELEASE_PLAN_ADMISSION"
+          value: '$(params.releasePlanAdmission)'
+      script: |
+        #!/bin/bash
+        set -e
+
+        EXODUS_ENV=$(get-resource "releaseplanadmission" "$(params.releasePlanAdmission)" "{.spec.data.exodus.env}")
+        output_file="/etc/exodus-rsync.conf"
+        
+        EXODUS_GW_URL="https://exodus-gw.corp.redhat.com"
+        EXODUS_GW_ENV="live"
+        EXODUS_GW_CERT="exodus-prod-cert.crt"
+        EXODUS_GW_KEY="exodus-prod-privatekey.pem"
+
+        # Check if EXODUS_ENV contains "cdn-qa"
+        if [[ "$EXODUS_ENV" == *"cdn-qa"* ]]; then
+          EXODUS_GW_CERT="exodus-stage-cert.crt"
+          EXODUS_GW_KEY="exodus-stage-privatekey.pem"
+          EXODUS_GW_URL="https://exodus-gw.corp.stage.redhat.com"
+        fi
+
+        # Check if EXODUS_ENV contains "cdn-stage"
+        if [[ "$EXODUS_ENV" == *"cdn-stage"* ]]; then
```

Comment:
So for cdn-qa, you will still use `EXODUS_GW_ENV=live`? Is that right?

Reply:
Yes, correct! for cdn-qa, we will be using the stage cert and key but env is still live

Reply:
OK, thanks for confirming.